lua/gluatest/loader.lua

```diff
@@ -29,20 +38,38 @@ function Loader.processFile( dir, fileName, groups )
     if not string.EndsWith( fileName, ".lua" ) then return end
 
     local filePath = dir .. "/" .. fileName
-    local fileOutput = include( filePath )
+    local success, result = pcall( function( givenFilePath )
+        local fileContent = file.Read( givenFilePath, "LUA" )
+        local compiled = CompileString( fileContent, "lua/" .. givenFilePath, false )
 
-    if not istable( fileOutput ) then
-        print( "GLuaTest: File " .. filePath .. " did not return a table - ignoring" )
-        return
+        if not isfunction( compiled ) then
+            return Loader.simpleError( compiled, givenFilePath )
```

Comment:
Is this error message sufficient for these situations? Would it be helpful to include any human language?